demo/src/lib/layout/playground/PlaygroundValue.svelte

```diff
@@ -2,13 +2,17 @@
 	import clearIcon from 'bootstrap-icons/icons/x-circle-fill.svg?raw';
 	import type {SingleValueContextApi} from './playground';
 	import Svg from '../Svg.svelte';
+	import type {SelectValues} from '@agnos-ui/common/propsValues';
 
 	export let type: string;
 	export let defaultValue: any;
 	export let api: SingleValueContextApi;
 	export let placeholder = '';
 	export let ariaLabel: string;
+	export let selectValues: SelectValues[] = [];
 
+	// used only for the select where the api. value = {value: [key]}
+	let value: string | undefined = api.isEmpty ? undefined : api.value.value;
```

Comment:
I think this logic should be managed in the api.

Reply:
I saw you changed this, thank you!
But I was thinking in a more complex change that should probably be done in a separate PR (to better use reactivity).